client/src/app/shared/video-playlist/video-add-to-playlist.component.html

```diff
@@ -59,7 +59,7 @@
     </div>
   </div>
 
-  <div class="new-playlist-button dropdown-item" (click)="openCreateBlock($event)" [hidden]="isNewPlaylistBlockOpened">
+  <div class="new-playlist-button dropdown-item" (click)="openCreateBlock($event)" [hidden]="isNewPlaylistBlockOpened" i18n>
```

Comment:
`i18n` basically adds the content of the element to the translation. To avoid confusing our translators as to what needs to be effectively translated, we try to avoid having code in our translatable input. Please wrap the text to translate with a `<span i18n> … </span>` instead.